sklearn/cluster/mean_shift_.py

```diff
@@ -103,8 +104,17 @@ def mean_shift(X, bandwidth=None, seeds=None, bin_seeding=False,
         Ignored if seeds argument is not None.
 
     min_bin_freq : int, optional
-       To speed up the algorithm, accept only those bins with at least
-       min_bin_freq points as seeds. If not defined, set to 1.
+        To speed up the algorithm, accept only those bins with at least
+        min_bin_freq points as seeds. If not defined, set to 1.
+
+    kernel : string, optional, default 'flat'
+        The kernel used to update the centroid candidates. Implemented kernels
+        are 'flat', 'rbf', 'epanechnikov' and 'biweight'.
+
+    gamma : float, optional, default 1.0
```

Comment:
I am a bit uncertain if we should also use bandwidth for rbf... not sure. In all other places we use gamma, I guess.